src/Fmt_ast.ml

```diff
@@ -352,6 +352,16 @@ let wrap_record c =
   if c.conf.space_around_collection_expressions then wrap "{ " "@ }"
   else wrap_fits_breaks c.conf "{" "}"
 
+let wrap_tuple ~parens ~no_parens_if_break c =
+  let wrap_fits_breaks_if _c cnd pre suf k fs =
+    fits_breaks_if cnd pre (pre ^ " ") fs ;
+    k fs ;
+    fits_breaks_if cnd suf ("@ " ^ suf) fs
+  in
+  if parens then wrap_fits_breaks_if c.conf true "(" ")"
```

Comment:
Why did you rewrite `wrap_fits_breaks_if` ? The one from `Fmt` should work fine here.